homeassistant/components/switchbot/switch.py

```diff
@@ -37,8 +37,8 @@ class SwitchBot(SwitchEntity, RestoreEntity):
     def __init__(self, mac, name, password) -> None:
         """Initialize the Switchbot."""
 
-        self._state = None
-        self._last_run_success = None
+        self._state: bool | None = None
```

Comment:
Why not change to using `_attr_state`? (The same could be done for some of the other properties here).

Reply:
I don't have a switchbot and the integration has no tests, so it wouldn't be possible for me to test that kind of change. 
It could also be out of the scope of this PR.
I do think _attr_ should be used here, but that's for a different PR by the code owner or a user.

> Why not change to using _attr_state?

I think you mean `_attr_is_on`, because you shouldn't change `_attr_state` for a ToggleEntity.
`_attr_is_on` cannot be set to None because it has a bool annotation in the ToggleEntitiy superclass, so that wouldn't fix this mypy issue.

https://github.com/home-assistant/core/blob/370799bd229c36bbb3428af4a20dcf4d8031faa2/homeassistant/helpers/entity.py#L867-L872